libtransmission/metainfo.cc

```diff
@@ -126,10 +126,10 @@ char* tr_metainfo_sanitize_path_component(char const* str, size_t len, bool* is_
         }
     }
 
-    for (size_t i = 0; i < TR_N_ELEMENTS(reserved_names); ++i)
+    for (auto const& reserved_name : ReservedNames)
     {
-        size_t const reserved_name_len = strlen(reserved_names[i]);
-        if (evutil_ascii_strncasecmp(ret, reserved_names[i], reserved_name_len) != 0 ||
+        size_t const reserved_name_len = std::size(reserved_name);
```

Comment:
I agree, it's possibly not needed; kept it because same length is referenced three times in two lines so it seemed like making a local temporary would be worthwhile. :man_shrugging: